src/components/SQForm/SQFormDropdown.tsx

```diff
@@ -112,15 +133,17 @@ function SQFormDropdown({
           onBlur={handleBlur}
           onChange={handleChange}
           labelId={labelID}
-          renderValue={renderValue}
+          renderValue={(value: unknown) =>
+            renderValue(value as Option['value'])
+          }
```

Comment:
[nit / suggestion]
Since you are defining renderValue on line 94 in this file. It might make sense to push these two functions together.

```
94. const renderValue = (value: unknown) {
  const optionValue = value as Option['value'];
  ...
  }
  ```
  
  Right now the function definition is kinda split and decreases readability.

Reply:
Good call. I did this to try to make TS happy. the `renderValue` prop expects a function who's first param is of `unknown` value. Even though it's entirely possible for us, the consumer, to know exactly what type the param will be. But there isn't a generic we can provide to tell the MUI component what the type is. So we have to cast it. Which kinda sucks. But it makes our _actual_ renderValue function type safe.

I'll smoosh em together for sure. There's another instance of this in another component, MultiSelect I beleive, i will change it there as well.